components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-controls/reporting-controls.component.html

```diff
@@ -41,6 +41,39 @@
             <chef-icon>help</chef-icon>
             <span>{{ summary.skipped.total | number }}</span>
           </span>
+          <span *ngIf="summary.waived.total > 0" class="control-result" [ngClass]="{'waived': summary.waived.total > 0}">
+            <div [attr.id]="'waived-icon' + control.id" class="waived-icon"></div>
+            <chef-tooltip
+              [attr.for]="'waived-icon' + control.id"
+              position="left">
+              <div class="control-waiver-details" *ngFor="let waiver of control.waivers">
+                <div class="waiver-details">
+                  <span class="label">Expires:</span>
+                  <span class="message">{{ waiver.expiration_date | datetime:'ddd, D MMMM YYYY HH:mm:ss [UTC]' }}</span>
```

Comment:
We actually have RFC2822 defined as a Constant DateTime helper here: https://github.com/chef/automate/blob/ce74f7cfe38018a32476cad92c479b4ac88a3b5f/components/automate-ui/src/app/helpers/datetime/datetime.ts#L6

Reply:
@SEAjamieD I've added DateTime, thanks for finding that! :)